packages/terra-avatar/docs/upgrade-guide.md

```diff
@@ -1,4 +1,36 @@
 # terra-avatar Upgrade Guide
+## Changes from version 2 to version 3
+
+### Removed
+* `Shared User` Variant From Avatar Variants
+
+### Added
+* `Generic` Variant with props same as `Shared User` and with `Variant` prop to support sub-variants like `single-user`, `shared-user` and `provider`.
```

Comment:
```suggestion
* the `generic` subcomponent that replaces the`sharedUser` subcomponent with a new `variant` prop that can be `single-user`, `shared-user`, or `provider`.
```

Reply:
Thank you @nramamurth. yes that sounds more appropriate. updated here : 5f171357be04821b6b25ee4000e814fe3f5bfaca